### ISSUE TITLE
Allow position of legend to be set with coordinate tuple as in Bokeh

### ISSUE THREAD

**Issue by rjp23** [pyviz/holoviews#3505]
Bokeh: https://bokeh.pydata.org/en/latest/docs/user_guide/styling.html#location

Legend positions can be set by position names but also by: a (x, y) tuple indicating an absolute location in screen coordinates (pixels from the bottom-left corner).

Holoviews supports the named-positions but not the coordinates.

**Comment by wmayner** [comment]
Incidentally, I can't find out how to change the position of a legend (even with just the position name) anywhere in the Holoviews documentation. Legends could be documented better, IMHO.

**Comment by philippjfr** [comment]
Agreed, I believe there should be a section about legends in the [Customizing Plots](http://holoviews.org/user_guide/Customizing_Plots.html) user guide. A PR to add that would be greatly appreciated, it should also cover the ``legend_offset`` parameter which does allow adjusting the legend in screen space.

**Comment by wmayner** [comment]
I'd be happy to submit a PR, but I don't know how legends work. A bit of a catch-22 😅

**Comment by zertrin** [comment]
For those landing here after googling for this, the position of the legend can be set with `.opts(legend_position='top_left')` for example.
